msal/tests/dev apps/XForms/XForms.Android/MainActivity.cs

```diff
@@ -47,13 +47,13 @@ protected override void OnCreate(Bundle bundle)
             global::Xamarin.Forms.Forms.Init(this, bundle);
             LoadApplication(new App());
 
-            App.UIParent = new UIParent(Xamarin.Forms.Forms.Context as Activity);
+            App.UIParent = new UIParent(this);
 
             #region Web browsers for MSAL.NET Android
             // To activate embedded webview, remove '//' from line 56 below, 
             // and comment out line 50 above -> App.UIParent = new UIParent(Xamarin.Forms.Forms.Context as Activity);
 
-            //App.UIParent = new UIParent(Xamarin.Forms.Forms.Context as Activity, true);
+            //App.UIParent = new UIParent(this, true);
```

Comment:
This is why dead code is difficult to maintain...

Reply:
This is the test app. And for different scenarios you would comment/uncomment. To Bogdans comment: Making some ui that switches between the scenarios instead of this? It will require some changes to the app but perhaps it's worth it... 

---
In reply to: [194104305](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1082#discussion_r194104305) [](ancestors = 194104305)